bin/build-metadata.js

```diff
@@ -331,24 +331,20 @@ function insertCountryAliases( data ) {
 }
 
 /**
- * Converts a given object to a JS string, then compiles and executes the given string and compares the two with deep
- * equality. If they are deeply equal, returns the JS string.
- * @param {{}} data
- * @returns {string}
+ * Wraps and saves data to '../client/components/phone-input/data.js'
+ * @param {Object} data
  */
-function convertToJSStringAndVerify( data ) {
-	const dataString = toJSString( data );
-	const parsedData = parseJSStringToData( dataString );
```

Comment:
`parseJSStringToData` can be removed too, it seems? And then `toJSString` and `objectKeyToJSString` too ;)

Reply:
removed in 1898bebbd